selfdrive/car/tests/test_models.py

```diff
@@ -104,6 +106,11 @@ def setUpClass(cls):
 
     cls.can_msgs = sorted(can_msgs, key=lambda msg: msg.logMonoTime)
 
+    # Sets enableDsu for cars with missing carFw
+    # TODO: get new route for TOYOTA.LEXUS_CTH
+    if len(car_fw) == 0 and CP is not None and CP.enableDsu:
+      car_fw = [car.CarParams.CarFw(ecu=car.CarParams.Ecu.unknown)]
+
```

Comment:
@adeebshihadeh should we dashcam the Lexus CTH? There's only one user in the last 90 days running a fork on a comma two